src/api/controllers/admin/admin.teams.js

```diff
@@ -18,11 +18,15 @@ module.exports = app => {
     const { User, Team } = req.app.locals.models
 
     try {
-      let teams = await Team.findAll({ include: [User] })
+      let teams = await Team.findAll({
+        include: [User]
```

Comment:
Ici tu peux directement filtrer les attributs dont tu as besoin plutôt que de tout récupérer : économie de temps et plus facile à maintenir.

```suggestion
     include: [User],
     attributes: [//liste des attributs nécessaires] 
   })
      
```

Reply:
Ok je savais pas je vais modifier tout ça

Reply:
C'est super pratique mais au final le fichier n'était jamais appelé donc je le supprime